### PR TITLE
Add more docs based on proposed wishlist

### PR DESCRIPTION
**Pull request by fjy**
Some of the items originally discussed here:
https://groups.google.com/forum/#!topic/druid-development/skWOuYETk7g

### CODE REVIEW

File: docs/content/operations/other-hadoop.md [hunk 1]
```diff
@@ -13,6 +13,12 @@ The Hadoop Index Task takes this parameter has part of the task JSON and the sta
 
 If you are still having problems, include all relevant hadoop jars at the beginning of the classpath of your indexing or historical nodes.
 
+Working with CDH
+----------------
+Members of the community have reported dependency conflicts between the version of Jackson used in CDH and Druid. Currently, our best workaround is to edit Druid's pom.xml dependencies to match the version of Jackson in your hadoop version and recompile Druid.
```
Comment (drcrallen) [inline docs/content/operations/other-hadoop.md]: I don't know of any specific patches at the moment, but there have been times in the past where we have found Jackson bugs and had them patched in mainline Jackson. I think they all have workarounds currently, though.

File: docs/content/querying/sql.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+layout: doc_page
+---
+# SQL Support for Druid
```
Comment (xvrl) [inline docs/content/querying/sql.md]: do we need a separate page for that? Maybe put it as part of the main querying page?

Comment (fjy) [inline docs/content/querying/sql.md]: I want to add more to this page in the future. I think it is important it has its own section

File: docs/content/operations/other-hadoop.md (hunk 1 above)
Comment (fjy) [inline docs/content/operations/other-hadoop.md]: I'm basing this based on numerous feedback in the forums about jackson conflicts with CDH

File: docs/content/misc/evaluate.md [hunk 3]
```diff
@@ -31,3 +31,20 @@ some [example specs](../configuration/production-cluster.html) for hardware for
...
+
+It is recommended you follow the [example production configuration](../configuration/production-cluster.html) for an actual production setup.
+
+1. node1: [Coordinator](../design/coordinator.html) + metadata store + zookeeper
+2. node2: [Broker](../design/broker.html) + [Historical](../design/historical.html)
+3. node3: [Overlord](../design/indexing-service.html)
+
+The coordination pieces (coordinator, metadata store, ZK) can be colocated on the same node. These processes do not require many resources, even for reasonably large clusters.
+ 
+The query pieces (broker + historical) can be colocated. You can add more of these nodes if your data doesn't fit on a single machine. Make sure to allocate enough heap/off-heap size to both processes.
```
Comment (nishantmonu51) [inline docs/content/misc/evaluate.md]: may be also add If running on a single node, you can directly send queries to the historical and skip running broker.

**Comment by gianm** [comment]
:+1:

**Comment by nishantmonu51** [comment]
+1

**Pull request merged**